qiskit_machine_learning/neural_networks/two_layer_qnn.py

```diff
@@ -81,22 +105,83 @@ def __init__(
         self._circuit.append(self._ansatz, range(num_qubits))
 
         # construct observable
-        self.observable = (
+        self._observable = (
             observable if observable is not None else PauliSumOp.from_list([("Z" * num_qubits, 1)])
         )
```

Comment:
Do you think we need such a class going forwards? Could instead we just stay with EstimatorQNN and have some convenience/factory function that given a pair of circuits/num qubits etc just builds out an EstimatorQNN and returns it? As far as can tell it just combines the pair into a single circuit and takes the input and weights from each of the circuits. Do we need a class for this? With some factory method the parameters could be ordered more comparably to EstimatorQNN. If a class was really wanted could we come up with some other name so it could be built out to avoid "combining" it with the old